conf/templates/handlebarswrapper.txt

```diff
@@ -27,7 +27,7 @@ context['_hb'] = Handlebars;
 <%= contents %>;
 
 const autoinit = function() {
-  if (window.ANSWERS) {
+  if (window.ANSWERS && window.ANSWERS.templates) {
```

Comment:
more of a question than a suggestion but do we prefer to stay away from optional chaining?

Reply:
We don't have a strong preference either way, however, we use it a lot in the core library